pkg/container/init.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package container contains helpers for working with container engines
+// This package has no stability guarantees whatsoever!
+package container
+
+import (
+	"os/exec"
+)
+
+//Engine is the container engine to use to launch containers
+var Engine string
```

Comment:
can we document the initialization behavior IE something like "by default this will be the first of podman or docker to be found in $PATH, with a fallback to docker" ?

We should probably also note supported values.

Reply:
Improved the comment.

Reply:
thanks